src/ReTestItems.jl

```diff
@@ -253,13 +259,14 @@ function runtests(
     nworkers = max(0, nworkers)
     retries = max(0, retries)
     timeout = ceil(Int, testitem_timeout)
+    timeout_profile_wait = ceil(Int, timeout_profile_wait)
```

Comment:
This will throw an `InexactError` if you pass `Inf`. Maybe that's fine?

Reply:
Yeah `Inf`s, `NaN`s and negative values will throw at some point, but I don't think this is an issue in practice.

Reply:
I've added a check for the negative numbers as we do the same thing for `timeout`